articles/dna-computing/index.md

```diff
@@ -0,0 +1,50 @@
+A good experience with biology and computer science is required to build algorithms to be executed in DNA computing. The information or data instead of being stored in binary digits will now be stored in the form of the bases A, T, G, C. The ability to synthesize short sequences of DNA artificially makes it possible to use these sequences as inputs for algorithms.
+
+
+### Applications
+The first theory of DNA computation was proposed by Leonard Adleman in 1994. He put his experimental theory into the test with a seven-point Hamiltonian path problem or also called the traveling salesman problem. The salesman in this problem needs to find the shortest path between seven cities whose distances are known in such a way that he crosses no city twice and returns to the original city. Adleman represented each city with a short DNA sequence with about 20 bases and a [complementary strand](https://en.wikipedia.org/wiki/Complementary_DNA) with 20 bases as the street between the cities. All the fragments are capable of linking with each other. When the fragments were put in a tube and mixed, the natural bonding tendencies the DNA created 109 formations or solutions in less than a second. Not all were correct and he took a week to extrapolate and filter out the shortest path through various techniques.
+
+![Image source](/engineering-education/dna-computing/image1.png)
+
+[_Image source_](https://www.cs.uaf.edu/2010/fall/cs441/proj1/dna/DNAComputingHTMLNotes.html)
+
+Though this solution was not ideal, this demonstration opened floodgates to a wide range of possibilities and applications. Few applications being  developed are mentioned below
+
+#### **Security**
+Deploying DNA algorithms in cryptography to build an [intrusion detection model](https://www.geeksforgeeks.org/intrusion-detection-system-ids/#:~:text=Dropping%20in%20Routers-,Intrusion%20Detection%20System%20(IDS),harmful%20activity%20or%20policy%20breaching.) is the most recent development. The ability to store 108 terabytes of data in 1 gram of DNA has led to the potential holding a huge one time pad. Another example is DNA steganography, in which a novel method was used to hide the messages in a microdot. Instead of the traditional binary encoding, each letter was denoted by three chemical bases i.e. the letter A was encoded by CGA. These messages are then encoded into DNA sequences and concealed by mixing it in a tube with a large amount of sonicated random human DNA. This led to the formation of microdots, which was then decoded by the receiver with appropriate primers(short sequence with complementary bases). However such encryption techniques have been posed with problems. The lack of a theoretical basis to explain the implementation and come up with good schemes seem to be a challenge. These are also expensive to apply, and analyze which requires modern infrastructure.
```

Comment:
Links to papers or articles on DNA OTP & stenography?

Reply:
The text highlighting in the link to geeksforgeeks does not work, I don't believe that site supports that.